plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -467,12 +467,9 @@ public boolean imageExists(String image) {
 		if (dockerConnection == null || StringUtils.isBlank(image)) {
 			return false;
 		}
-		String[] imageDetails = image.split(":");
-		if (imageDetails.length > 2) {
-			return false;
-		}
-		String repo = imageDetails[0];
-		String tag = (imageDetails.length == 1)?"latest":imageDetails[1];
+		DockerImageURI imageName = new DockerImageURI(image);
+		String repo =  imageName.getUriWithoutTag();
+		String tag = imageName.getTag() == null?"latest":imageName.getTag();
```

Comment:
I believe it will never be null https://github.com/openshift/openshift-restclient-java/blob/master/src/main/java/com/openshift/restclient/images/DockerImageURI.java#L61  but an alternate more expressive is `StringUtils.defaultIfEmpty(imageName.getTag(), "latest")`  or it could be` #defaultIfBlank()`, I never remember which one.